legion/tests/test_utils_external_file.py

```diff
@@ -136,11 +146,14 @@ def test_external_file_save_local(self):
         self.assertTrue(os.path.exists(target_file_path))
 
     def test_external_file_save_http_bin(self):
-        target_resource = 'https://httpbin.org/put'
+        httpbin_image = "kennethreitz/httpbin"
+        LOGGER.info('Starting httpbin container from {} image...'.format(httpbin_image))
```

Comment:
I think it will be better to move this log line to context manager

Reply:
Why? The container will start by the time one enter the context. So, there is almost no point in logging of the event then....

Reply:
@akharlamov , because of the fact context manager should be reusable. Message `starting container` should be printed from enter function of context manager, as I think.